pkg/client/generated/management/v3/zz_generated_resource_quota_limit.go

```diff
@@ -18,17 +19,18 @@ const (
 )
 
 type ResourceQuotaLimit struct {
-	ConfigMaps             string `json:"configMaps,omitempty" yaml:"configMaps,omitempty"`
-	LimitsCPU              string `json:"limitsCpu,omitempty" yaml:"limitsCpu,omitempty"`
-	LimitsMemory           string `json:"limitsMemory,omitempty" yaml:"limitsMemory,omitempty"`
-	PersistentVolumeClaims string `json:"persistentVolumeClaims,omitempty" yaml:"persistentVolumeClaims,omitempty"`
-	Pods                   string `json:"pods,omitempty" yaml:"pods,omitempty"`
-	ReplicationControllers string `json:"replicationControllers,omitempty" yaml:"replicationControllers,omitempty"`
-	RequestsCPU            string `json:"requestsCpu,omitempty" yaml:"requestsCpu,omitempty"`
-	RequestsMemory         string `json:"requestsMemory,omitempty" yaml:"requestsMemory,omitempty"`
-	RequestsStorage        string `json:"requestsStorage,omitempty" yaml:"requestsStorage,omitempty"`
-	Secrets                string `json:"secrets,omitempty" yaml:"secrets,omitempty"`
-	Services               string `json:"services,omitempty" yaml:"services,omitempty"`
-	ServicesLoadBalancers  string `json:"servicesLoadBalancers,omitempty" yaml:"servicesLoadBalancers,omitempty"`
-	ServicesNodePorts      string `json:"servicesNodePorts,omitempty" yaml:"servicesNodePorts,omitempty"`
+	AnyOther               map[string]string `json:"anyOther,omitempty" yaml:"anyOther,omitempty"`
```

Comment:
I'm not sure about the naming here, does `AnyOther` is a maybe a bit vague?

How about `AdditionalResources` or `AdditionalQuotas` or something similar?

Reply:
Looking at the Kubernetes naming https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#extended-resources maybe `ExtendedResources` would fit better?

Reply:
Done with commit cff4a96fd62eb0543360dc8f594d8743b13ce1f9 and preceding 2.